src/MoBi.Assets/AppConstants.cs

```diff
@@ -2018,6 +2013,11 @@ public static string BuildingBlockAlreadyContains(IReadOnlyList<string> paths)
 
             return sb.ToString();
          }
+
+         public static string ThisWillConvertPkSimModuleToExtensionModule(string moduleName)
+         {
+            return $"This will convert module '{moduleName}' to an extension module {Environment.NewLine} Continue?";
```

Comment:
would add "PK-Sim module" to the description + dot at the end
"This will convert **PK-Sim** module '{moduleName}' to an extension module {Environment.NewLine}**.** Continue?"